### PR TITLE
Add `draft` and `prerelease` options

### LINKED ISSUE #31: Add draft and prerelease option to YAML config
Hey fregante,

I would like to request to add **draft and prerelease option** as in many other _create release_ Github Actions.

### PR DESCRIPTION
**Pull request by BelKed**
Resolves #31.

- [x] Add `draft` and `prerelease` option to `index.js`
- [x] Update `readme.md`
- [x] Update `action.yml`

### CODE REVIEW

**Comment by BelKed** [comment]
@fregante, please, could you review?

**Comment by BelKed** [comment]
@fregante, please, could you review?

File: index.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ async function run() {
 		const exclude = core.getInput('exclude');
 		const dateFormat = core.getInput('date-format');
 		const reverseSort = core.getInput('reverse-sort');
+		const draftOption = core.getInput('draft') === 'true';
+		const prereleaseOption = core.getInput('prerelease') === 'true';
```
Comment (notlmn) [inline index.js]:
```suggestion
		const isDraft = core.getInput('draft') === 'true';
		const isPrerelease = core.getInput('prerelease') === 'true';
```

And change the respective instances of these variables down below too.

**Review by notlmn — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (BelKed) [reply index.js]: OK

**Review by notlmn — APPROVED** [review]
LGTM. @fregante might want to change some wording in readme.

**Comment by BelKed** [comment]
@fregante, please, could you review?

**Review by fregante — APPROVED** [review]

**Comment by Auburn** [comment]
Could you tag a new version that has this change please ♥️

**Comment by fregante** [comment]
Done!

**Pull request merged**